src/main/kotlin/com/moshbit/katerbase/MongoDatabase.kt

```diff
@@ -139,8 +141,9 @@ open class MongoDatabase(
       clientSettings = clientSettings
     )
 
+    val readChangeStreamFromSecondary = System.getenv("KATERBASE_READ_CHANGESTREAM_FROM_SECONDARY") == "true"
```

Comment:
imho should this be a parameter for class constructor, not an env var. because all other options are also parameters. and you even might want to have different values for different db connections